hkmc2/shared/src/test/mlscript-compile/Rendering.mjs

```diff
@@ -105,9 +105,9 @@ let Rendering1;
       tmp = Rendering.fold(lambda);
       tmp1 = Rendering.interleave(", ");
       tmp2 = Rendering.map(Rendering.render);
-      tmp3 = tmp2(...arg);
-      tmp4 = tmp1(...tmp3);
-      return tmp("[", ...tmp4, "]")
+      tmp3 = runtime.safeCall(tmp2(...arg));
```

Comment:
Oh noes why did these come back?

Reply:
I did some funny thing to the symbols. Let me fix it later